app/views/idv/shared/_document_capture.html.erb

```diff
@@ -60,4 +60,4 @@
         ) %>
   <% end %>
 
-<%= javascript_packs_tag_once 'document-capture' %>
+<%= javascript_packs_tag_once 'document-capture' %>
```

Comment:
Can you add the additional line back in?  No need to have this file show up in the diff list